packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.tsx

```diff
@@ -90,14 +101,18 @@ export class MessageBar extends React.Component<IMessageBarProps, IMessageBarSta
 
   private _renderMultiLine(): React.ReactElement<React.HTMLProps<HTMLAreaElement>> {
     return (
-      <div className={ this._getClassName() + ' ms-MessageBar-multiline' } role='status' aria-live='polite' aria-controls='ms-MessageBar-text'>
+      <div
+        className={ this._getClassName() + ' ms-MessageBar-multiline' }
+        role='status'
+        aria-live={ this._getAnnoucementPriority() }
+        aria-controls='ms-MessageBar-text'>
```

Comment:
aria-controls isn't needed. That attribute is used when one action on a page modifies the behavior of a different section of the page (like tabs and a tab panel, or collapsible regions).